client/me/help/help-results/index.jsx

```diff
@@ -24,7 +39,11 @@ module.exports = React.createClass( {
 		return (
 			<div className="help-results">
 				<SectionHeader label={ this.props.header }/>
-				{ this.props.helpLinks.map( helpLink => <HelpResult key={ helpLink.link } helpLink={ helpLink } iconTypeDescription={ this.props.iconTypeDescription } /> ) }
+				{ this.props.helpLinks.map( helpLink =>
+					<HelpResult
+						key={ helpLink.link } helpLink={ helpLink } iconTypeDescription={ this.props.iconTypeDescription }
+						onClick={ this.props.onClick ? this.createOnClick( this.props.onClick, helpLink ) : undefined } /> )
```

Comment:
Thanks, I wasn't sure how this would affect the pure rendering stuff, I'll make this change.